vllm/engine/llm_engine.py

```diff
@@ -1110,6 +1110,8 @@ def update_prefill_num_computed_tokens(
                 update_prefill_num_computed_tokens(seq_group, seq_group_meta,
                                                    len(output),
                                                    is_first_step_output)
+            elif not is_async:
+                seq_group.update_num_computed_tokens(1)
```

Comment:
There might be an issue here with multi-step + eager-mode - lets say there are 8 steps in a multi-step, when all the 8 outputs are submitted in a burst, process_model_outputs should account for all the 8 steps, but here we seem to account for only one ?